modules/dev-utils/src/main/java/org/apache/ignite/development/utils/IgniteWalConverter.java

```diff
@@ -110,4 +135,29 @@ public static void main(String[] args) throws Exception {
         if (stat != null)
             System.out.println("Statistic collected:\n" + stat.toString());
     }
+
+    /**
+     * Converting {@link WALRecord} to a string with sensitive data.
+     *
+     * @param walRecord Instance of {@link WALRecord}.
+     * @param sensitiveData Strategy for processing of sensitive data.
+     * @return String representation of {@link WALRecord}.
+     */
+    private static String toString(WALRecord walRecord, ProcessSensitiveData sensitiveData) {
+        if (SHOW == sensitiveData || HIDE == sensitiveData)
+            return walRecord.toString();
+
+        if (MetastoreDataRecord.class.isInstance(walRecord))
```

Comment:
Same here, why not "instanceof"?

Reply:
There's no overhead, it's just my style.